packages/arb-validator/coordinator/rpcserver.go

```diff
@@ -40,7 +41,12 @@ func NewRPCServer(
 	connectionInfo ethbridge.ArbAddresses,
 	ethURL string,
 ) *RPCServer {
-	return &RPCServer{NewServer(machine, key, validators, connectionInfo, ethURL)}
+	return StartRPCServerVM(NewCoordinator(machine, key, validators, connectionInfo, ethURL))
```

Comment:
Can you explain the change from `NewServer()` to `StartRPCServerVM(NewCoordinator())`?